### PR TITLE
[ko] Update Korean continuous-delivery.md naturally

### PR DESCRIPTION
**Pull request by seokho-son**
### Describe your changes
Considering the original English document, update Korean document naturally.
https://github.com/cncf/glossary/edit/main/content/en/continuous-delivery.md

 - 원문과 조금 다른 의미 및 문장 개선 등

### Related issue number or link (ex: `resolves #issue-number`)
None

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ko/continuous-delivery.md [hunk 1]
```diff
@@ -7,31 +7,31 @@ tags: ["방법론", "애플리케이션", ""]
...
 필요한 경우 변경사항을 롤백(rollback)하는 방법을 제공한다.
-지속적인 통합(Continuous integration, CI)은 지속적인 전달에 필요한 선결 단계이다(즉,
-변경사항이 테스트 및 배포되려면 그 전에 문제 없이 병합되어야 한다).
+지속적인 통합(continuous integration, CI)은 지속적인 전달에 필요한 선결 단계이다. (즉,
+변경사항이 테스트 및 배포되려면 그 전에 문제 없이 병합되어야 한다.)
 
 ## 다루는 문제
 
-[신뢰할 수 있는](/reliability/) 업데이트 배포는 대규모일 때 문제가 된다.
+업데이트 배포의 [신뢰성](/reliability/)은 대규모일 때 더 큰 문제가 된다.
```
Comment (yunkon-kim) [inline content/ko/continuous-delivery.md:20]:
```suggestion
업데이트 배포의 [신뢰성](/reliability/)은 규모에 따라 문제가 된다.
```

"대규모"의 의미가 애매하기도하고, 원문도 이를 직접 적으로 나타내고 있지 않아서 수정 의견 달아봤습니다 ^^
> Deploying [reliable](https://github.com/cncf/glossary/blob/main/reliability) updates becomes a problem at scale.

**Review by yunkon-kim — APPROVED** [review]
LGTM ^^

File: content/ko/continuous-delivery.md (hunk 1 above)
Comment (seokho-son) [reply content/ko/continuous-delivery.md:20]: @yunkon-kim 리뷰 감사합니다. 
at scale이 문맥상으로는 (at: 특정할 수 있을 정도의 큰) scale로 
규모가 커진 상황을 강조하는 표현인 듯 하여, 살짝 의역한 것이었습니다.

Comment (seokho-son) [reply content/ko/continuous-delivery.md]: 제안해주신 내용으로 반영하였습니다! :)

**Review by jihoon-seo — APPROVED** [review]
LGTM 입니다~~

**Pull request merged**